scripts/zones/Temenos/bcnms/central_temenos_3rd_floor.lua

```diff
@@ -2,37 +2,43 @@
 -- Area: Temenos
 -- Name: Central Temenos 3rd Floor
 -----------------------------------
-require("scripts/globals/limbus");
+require("scripts/globals/limbus")
 require("scripts/globals/battlefield")
-require("scripts/globals/keyitems");
+require("scripts/globals/keyitems")
+local ID = require("scripts/zones/Temenos/IDs")
+
+function onBattlefieldInitialise(battlefield)
+    battlefield:setLocalVar("loot", 1)
+    SetServerVariable("[Central_Temenos_3rd_Floor]Time", battlefield:getTimeLimit()/60)
+    tpz.limbus.handleDoors(battlefield)
+    tpz.limbus.setupArmouryCrates(battlefield:getID())
+    DespawnMob(ID.mob.TEMENOS_C_MOB[3]+3)
+    DespawnMob(ID.mob.TEMENOS_C_MOB[3]+11)
```

Comment:
To show I _have_ been paying attention: These are for the Mook Orc DRG and the Boss Yagudo SMN, right?

Reply:
Yes